language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/codegen/InterfaceCompanion.java

```diff
@@ -19,4 +25,11 @@ protected InterfaceCompanion(Identifier templateId, ValueDecoder<View> valueDeco
     super(templateId);
     this.valueDecoder = valueDecoder;
   }
+
+  @Override
+  public TransactionFilter transactionFilter(Set<String> parties) {
+    Filter filter = new InclusiveFilter(Collections.emptySet(), Map.of(TEMPLATE_ID, Filter.Interface.INCLUDE_VIEW));
```

Comment:
Does it makes sense to always include view or maybe we should pass in a method argument here and let the caller decide

Reply:
If you don't include the view, the decoding part of the utility is guaranteed to fail to find the view. This utility should work in harmony with the other utilities that contribute to the goal of typed ACS/transaction streams.